L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -38,11 +38,11 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
         // Swipe to the second screen
         app.buttons["\(rootA11yId)PrimaryButton"].tap()
         currentScreen += 1
-        waitForExistence(app.buttons["\(rootA11yId)PrimaryButton"])
+        waitForExistence(app.buttons["SignSyncCardSecondaryButton"])
         snapshot("Onboarding-2")
 
         // Swipe to the third screen
-        app.buttons["\(rootA11yId)SecondaryButton"].tap()
+        app.buttons["SignSyncCardSecondaryButton"].tap()
```

Comment:
Actually I have just tested that the test works but with this click we go to the Homescreen.
So, I suggest, either remove this part (from line 44), or rename the Snapshot in L47 as "Homescreen first visit"